### ISSUE TITLE
Overlapping Text when viewed in Chrome Browser

### ISSUE THREAD

**Issue by f0rtyseven** [fedora-infra/apps.fp.o#41]
The Diagram is not fitting the whole window in Chrome Browser on my Windows 8.1 system. Also when I click on some links to zoom in, like QA, Infrastructure etc, the texts on the end nodes become overlapped, making them unreadable. Can provide screenshots if required.

**Comment by pypingou** [comment]
The offered screenshots might be helpful if you can :)

**Comment by f0rtyseven** [comment]
Here are the screenshots(4 of them). Can upload more if needed.
http://imgur.com/a/yxqLz

**Comment by pypingou** [comment]
Why not upload them directly to the ticket here?

**Comment by f0rtyseven** [comment]
Sorry. Didn't know about the norms here. I'm new to this. I was on Reddit previously and that was the norm. Sorry for the inconvenience caused.
![1 - rw8inpl](https://cloud.githubusercontent.com/assets/14765535/12063927/2687a04c-afdd-11e5-9e4c-8d2b4cfb8e96.jpg)
![2 - jfakvwf](https://cloud.githubusercontent.com/assets/14765535/12063928/26917270-afdd-11e5-86e9-d1871c43b1b1.jpg)
![3 - ubtwyz8](https://cloud.githubusercontent.com/assets/14765535/12063930/26a8724a-afdd-11e5-92b0-6dce97f2ab03.jpg)
![4 - ewsphlr](https://cloud.githubusercontent.com/assets/14765535/12063929/26a660cc-afdd-11e5-8b0f-c0c05aac86c9.jpg)

**Comment by ralphbean** [comment]
I agree this is a problem, but I'm not sure what to do about this.  When we launched the page, we had less apps than we do now.  Things are getting pretty cluttered.  Perhaps we need a new design all together?

**Comment by f0rtyseven** [comment]
Personally, I loved the design. Though I'm not a master of coding but I believe some tweaking would be enough to make it look the way it is meant to look. I have something on my mind, but don't know how it'll be done through code. My suggestions would be,
1. Make the whole diagram on the homepage to appear a bit smaller so that it fits the screen, and if that makes the texts overlap, showing only the icons at the end nodes. The group texts like Infrastructure, QA etc. can appear big as they are not overlapping. When a user clicks on the group name, it fans out showing the full name of the icons.
2. Making the nodes appear a bit further apart when we click to zoom in on them
3. If no. 2 doesn't work then making the length of the connecting edges variable(i mean some of them bigger and some of them smaller) to avoid the overlapping of texts

I'm only trying to suggest here. If I knew the coding, this is how i'd try to overcome this problem.

**Comment by f0rtyseven** [comment]
Btw wish you a very Happy New Year. Its 11.59 pm here. :)

**Comment by ralphbean** [comment]
Thanks @f0rtyseven.   You too!

A thought just occurred to me that might help inspire a solution:  The text only ever overlaps at the top and bottom of the circle.  It never overlaps on the left and right sides of the circle.  This is because (at least our untranslated English text) always reads horizontally from left to right.

**Comment by Germano0** [comment]
Confirming the bug on Firefox 50. When you select a subtree, you will get also unexpanded graph areas that will go into not viewable areas of the screen

**Comment by Germano0** [comment]
In a few days I will do a public presentation of Fedora Apps. Is there a chance to get this bug fixed soon?
